views/lib/default.hbs

```diff
@@ -68,6 +68,10 @@
 </dl>
 	{{#block "body"}}{{/block}}
 
+	<!-- Mopinion Pastea.se start -->
+	<script type="text/javascript">(function(){var id="5l9dq4pf776dwq8nknaxjdyyalysbgs4q8g";var js=document.createElement("script");js.setAttribute("type","text/javascript");js.setAttribute("src","deploy.mopinion.com/js/pastease.js");js.async=true;document.getElementsByTagName("head")[0].appendChild(js);var t=setInterval(function(){try{new Pastease.load(id);clearInterval(t)}catch(e){}},50)})();</script>
+	<!-- Mopinion Pastea.se end -->
+
 </body>
```

Comment:
This should be behind a feature flipper, also this will require changed CORS Settings (that must be handled when this is active). As we are trying to minimize changes in this phase of lock down this work should be postponed and transferred to the core teams for finishing.